docs/updating/update-to-43.md

```diff
@@ -16,8 +16,111 @@ modified_at: 2024-07-31
 
 ## Update to CKEditor&nbsp;5 v43.0.0
 
-_Released on August xx, 2024._
+_Released on August 7, 2024._
 
 For the entire list of changes introduced in version 43.0.0, see the [release notes for CKEditor&nbsp;5 v43.0.0](https://github.com/ckeditor/ckeditor5/releases/tag/v43.0.0).
 
-Below are the most important changes that require your attention when upgrading to CKEditor&nbsp;5 v43.0.0.
\ No newline at end of file
+Below are the most important changes that require your attention when upgrading to CKEditor&nbsp;5 v43.0.0.
+
+### Export to Word v2 becomes the default
+
+In CKEditor&nbsp;5 v42.0.0, we introduced a new version of Export to Word plugin that utilizes the updated converter (marked as V2). It was introduced in an opt-in manner. Starting with v43.0.0, the Export to Word V2 is **the default** configuration. This may require updates to the editor's configuration if you use this plugin.
+
+Example V1 configuration:
+
+```js
+exportWord: {
+			tokenUrl: 'https://example.com/cs-token-endpoint',
+			fileName: 'my-file.docx',
+			converterOptions: {
+				format: 'A4',
+				margin_top: '20mm',
+				margin_bottom: '20mm',
+				margin_right: '12mm',
+				margin_left: '12mm'
+			}
+		}
+```
+
+The same migrated to V2:
+
+```js
+exportWord: {
+			tokenUrl: 'https://example.com/cs-token-endpoint',
+			fileName: 'my-file.docx',
+			converterOptions: {
+				document: {
+					size: 'A4',
+					margin: {
+						top: '20mm',
+						bottom: '20mm',
+						right: '12mm',
+						left: '12mm'
+					}
+				}
+			}
+		}
+```
+
+You can find the full V2 configuration in the {@link module:export-word/exportword~ExportWordConverterOptionsV2} API documentation.
+
+Alongside with the configuration changes we decided to switch the default `auto_pagination` option to `false`. This option was only used with the pagination feature. When it is enabled the breaks in the pagination force the page breaks in the exported Word document. However, browser engines and Microsoft Word differ significantly. Because of that we decided to turn this option to `false`. If you still want to enforce breaks from pagination, change it in the configuration to `true`.
+
+### Typing bug fixes
```

Comment:
Seldom do we mention such stuff in the migration guide.

Reply:
We discussed on the daily :blush: